packages/preview/modernpro-coverletter/0.0.1/README.md

```diff
@@ -0,0 +1,64 @@
+# Typst-ModernPro-Coverletter
+
+This is a cover letter template for Typst with Sans font. It is a modern and professional cover letter template. It is easy to use and customize. This cover letter template is suitable for any job application or general purpose.
+
+## How to use
+
+### Use from the Typst Universe
+
+It is simple and easy to use this template from the Typst Universe. If you prefer to use local editor and `typst-cli`, you can use the following command to create a new cover letter project with this template.
+
+```bash
+typst init @preview/modernpro-coverletter
+```
+
+It will create a new cover letter project with this template in the current directory.
+
+### Use from GitHub
+
+You can also use this template from GitHub. You can clone this repository and use it as a normal project.
+
+```bash
+git clone https://github.com/jxpeng98/typst-coverletter.git
+```
+
+## Features
+
+The template will have the following features:
+
+```typst
+#show: main.with(
+  fonttype: "macfont",
+  name: "Exam Example",
+  address: [London, UK],
+  contacts: (
+    (text: "08856", link: ""),
+    (text: "example.com", link: "https://www.example.com"),
+    (text: "github.com", link: "https://www.github.com"),
+    (text: "123@example.com", link: "mailto:123@example.com"),
+  ),
+  recipient: (
+    starttitle: "Dear Committee Members,",
```

Comment:
kebab case, i.e. `start-title` would be more idiomatic here

Reply:
I change all double words variables into the kebab case, like `start-title`, `cl-title` and `font-type`.